### PR TITLE
[IMP] purchase_request: update existing line if any

### PR DESCRIPTION
**Pull request by mmequignon**
supersedes https://github.com/OCA/purchase-workflow/pull/1381

### CODE REVIEW

**Comment by mmequignon** [comment]
Test coming for this commit https://github.com/OCA/purchase-workflow/pull/1451/commits/531e95218f7c0ac0821807985dfe087c74132b97

**Comment by StefanRijnhart** [comment]
Can you squash commits?

File: purchase_request/models/stock_rule.py [hunk 1]
```diff
@@ -91,10 +91,17 @@ def _run_buy(self, procurements):
...
+        moves = procurement.values.get("move_dest_ids")
+        if moves:
+            return [m.id for m in moves]
+        return []
+
     def create_purchase_request(self, procurement_group):
         """
         Create a purchase request containing procurement order product.
         """
+        # import pdb; pdb.set_trace()
```
Comment (TDu) [inline purchase_request/models/stock_rule.py]:
```suggestion
```

**Review by TDu — APPROVED** [review]
LG

**Pull request merged**